src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+  const changeCss = sourceString.split(';');
+
+  changeCss.forEach((item) => {
+    const [key, value] = item.split(':');
+
+    if (value !== undefined) {
```

Comment:
```suggestion
    if (value) {
```
You don't need to compare with undefined, just check if value exists